src/orbbec/ob_k4a_impl.cpp

```diff
@@ -569,31 +595,50 @@ k4a_result_t init_device_context(k4a_device_t device_handle)
         }                                                                                                              \
     } while (0);
 
-void device_timestamp_sync_with_host(k4a_device_t device_handle, uint32_t intervalMicroseconds)
+void device_timestamp_sync_with_host(k4a_device_t device_handle, uint32_t interval_usec)
 {
     k4a_device_context_t *device_ctx = k4a_device_t_get_context(device_handle);
     ob_error *ob_err = NULL;
     ob_device_timer_sync_with_host(device_ctx->device, &ob_err);
     CHECK_OB_ERROR(&ob_err);
-    if(intervalMicroseconds == 0){
+    if(interval_usec == 0){
         return;
     }
 
     std::unique_lock<std::mutex> lck(device_ctx->clock_sync_mtx);
-    while(device_ctx->clock_sync_cv.wait_for(lck,std::chrono::microseconds(intervalMicroseconds)) == std::cv_status::timeout){
+    while(device_ctx->clock_sync_cv.wait_for(lck,std::chrono::microseconds(interval_usec)) == std::cv_status::timeout){
         if(device_ctx->current_device_clock_sync_mode == K4A_DEVICE_CLOCK_SYNC_MODE_SYNC){
             if(device_ctx->device){
                 ob_device_timer_sync_with_host(device_ctx->device, &ob_err);
             }
         }
     }
 }
+k4a_result_t k4a_set_soft_filter(k4a_device_t device_handle, bool filter_switch){
```

Comment:
Perhaps `k4a_result_t k4a_enable_soft_filter(k4a_device_t device_handle, bool enable)` could be a better option.